app/components/ClassCard.js

```diff
@@ -0,0 +1,58 @@
+"use client";
+import { balthazar } from "../lib/fonts";
+import Image from "next/image";
+import { useState } from "react";
+
+export default function ClassCard({
+  title,
+  imageUrl,
+  description,
+  pickLabel,
+  learnMoreLabel,
+  learnMoreInfo,
+  onPick,
+}) {
+  const [showMore, setShowMore] = useState(false);
+
+  const handleLearnMore = () => {
+    setShowMore(!showMore);
+  };
+  return (
+    <>
```

Comment:
Is there a reason to nest this inside a fragment?

Reply:
Thats how I create my components, less space taken up on the DOM, so i heard

Reply:
Oh i see i already have article so theres no need for it